tools/warning-to-plist/tests/unit/tidy_output_test_files/files/test3.cpp

```diff
@@ -0,0 +1,11 @@
+#include "test3.hh"
+
+int main(int argc, const char** /*argv*/) {
+  int* x = 0;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The scope of the variable 'x' can be reduced.](https://app.codacy.com/app/CodeChecker/codechecker/pullRequest?prid=4169881)